Flow.Launcher/ResultListBox.xaml

```diff
@@ -91,6 +91,7 @@
                     Value="True">
                     <Setter TargetName="Title" Property="Style" Value="{DynamicResource ItemTitleSelectedStyle}" />
                     <Setter TargetName="SubTitle" Property="Style" Value="{DynamicResource ItemSubTitleSelectedStyle}" />
+                    <Setter TargetName="imgIco" Property="Style" Value="{DynamicResource ItemImageSelectedStyle}" />
```

Comment:
Small nitpick: Maybe use the same naming style here as Title / Subtitle. ImageIcon instead of imgIco